engine/lib/metadata.php

```diff
@@ -91,6 +91,11 @@ function create_metadata($entity_guid, $name, $value, $value_type = '', $owner_g
  * @return bool
  */
 function update_metadata($id, $name, $value, $value_type, $owner_guid, $access_id) {
+	
+	if (strlen($value) >= 50) {
+		elgg_log(" ", 'WARNING');
```

Comment:
Replace `" "` with `"Metadata '{$name}' may be truncated by MySQL's TEXT column."`